app/views/categories/show.html.erb

```diff
@@ -8,9 +8,9 @@
     </div>
 
     <ul class="px-2">
-  <% @category.purchases.each do |purchase| %>
+  <% @purchases.each do |purchase| %>
    <a href="<%= category_purchase_path(@category, purchase) %>" >
-    <li class="py-3 bg-white flex justify-between px-2">
+    <li class="py-3 bg-white flex justify-between px-2 mb-2">
       <div>
         <p class="font-bold">
           <%= purchase.name %>
```

Comment:
- [ ] Great job s far, however., For a given category, the list of transactions should be presented in order by the most recent and the first please address this 👍🏻 
![image](https://github.com/migo1/Budget-App/assets/79058364/fa08a203-b675-42da-8625-a194311ace35)

*

Reply:
@BenMukebo noted, thanks for the review I'll work on the requested changes